src/Nethermind/Nethermind.Merge.Plugin/Synchronization/BeaconSync.cs

```diff
@@ -114,6 +115,11 @@ public bool IsBeaconSyncHeadersFinished()
             }
             return null;
         }
+
+        public Keccak GetFinalizedHash()
+        {
+            return _blockCacheService.FinalizedHash;
```

Comment:
This cannot go from BlockTree right? As before we sync it won't give us correct answer?

Reply:
`FinalizedHash` is needed in `MultiSyncModeSelector`. We could potentially pass it through `SyncProgressResolver`, but `BeaconSync` fits perfect for both, `MultiSyncModeSelector` and `PivotUpdator` (`BeaconSync` is needed in `PivotUpdator` anyway because of `GetTargetBlockHeight()`).